specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -211,6 +228,213 @@ union EvictionPolicy {
   Deallocate: "Deallocate",
 }
 
+@added(Versions.v2024_11_01)
+@doc("AdditionalCapabilities for VM.")
+model AdditionalCapabilities {
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "ultraSSDEnabled which does not follow camel casing"
+  @doc("The flag that enables or disables a capability to have one or more managed data disks with UltraSSD_LRS storage account type on the VM or VMSS. Managed disks with storage account type UltraSSD_LRS can be added to a virtual machine or virtual machine scale set only if this property is enabled.")
+  ultraSSDEnabled?: boolean;
+
+  @doc("The flag that enables or disables hibernation capability on the VM.")
+  hibernationEnabled?: boolean;
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "VMAttributeMinMax which does not follow camel casing"
+@added(Versions.v2024_11_01)
+@doc("While retrieving VMSizes from CRS, Min = 0 (uint.MinValue) if not specified, Max = 4294967295 (uint.MaxValue) if not specified. This allows to filter VMAttributes on all available VMSizes.")
+model VMAttributeMinMax {
+  @doc("Min VMSize from CRS, Min = 0 (uint.MinValue) if not specified.")
+  min?: int32;
+
+  @doc("Maz VMSize from CRS, Max = 4294967295 (uint.MaxValue) if not specified.")
+  max?: int32;
+}
+
+@added(Versions.v2024_11_01)
+@doc("VMSizes supported by Azure VMs. Included is a union of Excluded and Required.")
+union VMAttributeSupport {
+  string,
+
+  @doc("All VMSizes having the feature support will be excluded.")
+  Excluded: "Excluded",
+
+  @doc("SDD DiskType.")
+  Included: "Included",
+
+  @doc("VMSizes that have the feature support and that do not have the feature support will be used.")
+  Required: "Required",
+}
+
+@added(Versions.v2024_11_01)
+@doc("VMCategories defined for Azure VMs.")
+union VMCategory {
+  string,
+
+  @doc("GeneralPurpose VM.")
+  GeneralPurpose: "GeneralPurpose",
+
+  @doc("ComputeOptimized VM.")
+  ComputeOptimized: "ComputeOptimized",
+
+  @doc("MemoryOptimized VM.")
+  MemoryOptimized: "MemoryOptimized",
+
+  @doc("StorageOptimized VM.")
+  StorageOptimized: "StorageOptimized",
+
+  @doc("GpuAccelerated VM.")
+  GpuAccelerated: "GpuAccelerated",
+
+  @doc("FpgaAccelerated VM.")
+  FpgaAccelerated: "FpgaAccelerated",
+}
+
+@added(Versions.v2024_11_01)
+@doc("Architecture types supported by Azure VMs.")
+union CpuManufacturer {
+  string,
+
+  @doc("Intel CPU.")
+  Intel: "Intel",
+
+  @doc("AMD CPU.")
+  AMD: "AMD",
+}
+
+@added(Versions.v2024_11_01)
+@doc("Different kind of Local storage disk types supported by Azure VMs.")
+union LocalStorageDiskType {
+  string,
+
+  @doc("HDD DiskType.")
+  HDD: "HDD",
+
+  @doc("SDD DiskType.")
+  SSD: "SSD",
+}
+
+@added(Versions.v2024_11_01)
```

Comment:
ArchitectureType values are wrong. Please fix this. @rahuls-microsoft

Reply:
Updated.